src/components/order-service/order-service.controller.ts

```diff
@@ -72,7 +93,34 @@ export class OrderServiceController implements angular.IController {
     return;
   }
 
-  public cancelOrder() {
-    return;
+  public closePanel() {
+    if (angular.isFunction(this.ctrl.handleClose)) {
+      this.ctrl.handleClose();
+    }
+  }
+
+  private listProjects() {
+    this.DataService.list('projects', this.$scope).then((response: any) => {
+      this.ctrl.projects = _.sortBy(response.by('metadata.name'), this.$filter('displayName'));
+      this.ctrl.selectedProject = _.first(this.ctrl.projects);
+    });
+  }
+
+  private makeServiceInstance() {
+    let serviceClassName = _.get(this, 'ctrl.serviceClass.resource.metadata.name');
+    let serviceInstance = {
+      kind: 'Instance',
+      apiVersion: 'servicecatalog.k8s.io/v1alpha1',
+      metadata: {
+        namespace: this.ctrl.selectedProject.metadata.name,
+        generateName: serviceClassName + '-'
+      },
+      spec: {
```

Comment:
I'm not sure when we update this to handle async whether thats a URL param or if its a field on the object,  @pmorie ?  not seeing anything in the InstanceSpec in types.go